packages/protocol/test/bridge/libs/LibBridgeProcess.test.ts

```diff
@@ -55,17 +54,8 @@ describe("LibBridgeProcess", async function () {
             value: ethers.utils.parseEther("10.0"),
         });
 
-        libTrieLink = await (await ethers.getContractFactory("LibTrieProof"))
-            .connect(owner)
-            .deploy();
-        await libTrieLink.deployed();
-
         libProcessLink = await (
-            await ethers.getContractFactory("LibBridgeProcess", {
-                libraries: {
-                    LibTrieProof: libTrieLink.address,
-                },
-            })
+            await ethers.getContractFactory("LibBridgeProcess")
```

Comment:
Please ignore this file and all Bridge tests. These tests need complete rewrite.